com.microsoft.mrtk.graphicstools.unity/Samples~/MaterialGallery/Textures/simpleVoronoi_05.jpg.meta

```diff
@@ -0,0 +1,96 @@
+fileFormatVersion: 2
```

Comment:
Could you name this texture like the others? Maybe SimpleVoronoi_emission?

Reply:
Renamed to 'ShaderBall_emissive'